src/io/snapshot.cc

```diff
@@ -48,7 +48,7 @@ Snapshot::Snapshot(const std::string& prefix, Mode mode, int max_param_size /*in
     text_reader_ptr->Open(prefix + ".desc");
     std::string key, val;
     while (text_reader_ptr->Read(&key, &val)) {
-      if (key == "SINGA_VERSION")
+      if (key == "0")
```

Comment:
this is not the best to way fix the bug.
need to update the textfilereader and writer: https://github.com/apache/incubator-singa/blob/master/src/io/textfile_writer.cc#L52

Reply:
can figure out a way to retain if (key == "SINGA_VERSION")

Reply:
yes. it's a little bit strange. the code can pass in my local environment. 
i am trying to find the reason